pkgs/development/compilers/graalvm/default.nix

```
@@ -0,0 +1,20 @@
+{
+  lib,
+  stdenv,
+  callPackage,
+  fetchurl,
```

github actions failed here

```suggestion
  callPackage,
```